cypress/support/commands/drReset.js

```diff
@@ -0,0 +1,40 @@
+import s from '../../integration/dryRunner/selectors'
+
+const reset = ({ equals = false, modifier = 'NONE' }) => {
```

Comment:
```suggestion
const reset = ({ equals = false, modifier = 'NONE' } = {}) => {
```

This is so we don’t have to pass an empty object when we want the default options.

Reply:
Right otherwise () did not work 👍